src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -9052,6 +9078,20 @@ void Assembler::evpcmpb(KRegister kdst, KRegister mask, XMMRegister nds, Address
   emit_int8((unsigned char)comparison);
 }
 
+void Assembler::evpcmpub(KRegister kdst, KRegister mask, XMMRegister nds, XMMRegister src,
```

Comment:
Do you think it worthwhile to combine `evpcmpb` and `evpcmpub`? If not then do they need to differ as much as they do?

Reply:
Yes, I can look into combining those two.